build.yaml

```diff
@@ -3645,12 +3694,12 @@ steps:
           --filter 'tags.items=batch2-agent AND labels.namespace={{ default_ns.name }}' \
           --format="table[no-heading](zone.basename(), name)" \
           --project {{ global.gcp_project }} \
-        | xargs -n2 -r sh -c 'gcloud -q compute instances delete --zone "$1" --project {{ global.gcp_project }} "$2" || true' argv0
+        | xargs -P4 -n2 -r sh -c 'gcloud -q compute instances delete --zone "$1" --project {{ global.gcp_project }} "$2" || true' argv0
```

Comment:
This is probably fine, but have you tried doing, say, `P8`? I feel like we ought to be able to issue delete requests for all live instances at once and there could be many more than 4 across all the pools.